### PR TITLE
feat(checkout): CHECKOUT-6480 Offline payment method e2e tests

### PR DESCRIPTION
**Pull request by bc-peng**
## What?
Added two e2e tests to ensure all `offline` components of Checkout SDK/JS are working together.

Two tested payment methods are:
- Cash on delivery.
- Pay in store.

## Why?
Reduce the chance of introducing a regression.

## Testing / Proof
CI checks.

@bigcommerce/checkout

### CODE REVIEW

File: packages/payment-integration-test-framework/src/fixture/pageObject/playwright/senstiveDataConfig.ts [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 export const sensitiveHeaders = [
+    'accept',
+    'alt-svc',
     'authorization',
+    'cf-cache-status',
+    'cf-ray',
     'cookie',
+    'nel',
+    'referer',
+    'report-to',
```
Comment (bc-peng) [inline packages/payment-integration-test-framework/src/fixture/pageObject/playwright/senstiveDataConfig.ts:11]: Many new `cloudflare`-related headers appeared in the HAR file, which, in my opinion, revealed far too many tech details. 

Therefore, I updated the `sensitiveHeaders` config.

File: packages/payment-integration-test-framework/src/fixture/pageObject/playwright/senstiveDataConfig.ts [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
     'token',
-    'x-session-hash',
-    'x-xsrf-token',
-    'referer',
     'user-agent',
     'x-checkout-sdk-version',
-    'accept',
     'x-request-id',
+    'x-session-hash',
+    'x-xsrf-token',
```
Comment (bc-peng) [inline packages/payment-integration-test-framework/src/fixture/pageObject/playwright/senstiveDataConfig.ts:18]: Sort all attributes in ascending order as well.

File: packages/offline-payment-integration/e2e/OfflinePaymentMethod.spec.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {
+    PaymentStepAsGuestPreset,
+    test,
+} from '@bigcommerce/checkout/payment-integration-test-framework';
+
+test.describe('Offline payment method', () => {
+    test('`Pay in Store` payment method is working', async ({ assertions, checkout }) => {
+        // Testing environment setup
+        await checkout.use(new PaymentStepAsGuestPreset());
+        await checkout.start('Pay in Store');
```
Comment (animesh1987) [inline packages/offline-payment-integration/e2e/OfflinePaymentMethod.spec.ts:10]: Is this the text on the payment method field?

Comment (bc-peng) [reply packages/offline-payment-integration/e2e/OfflinePaymentMethod.spec.ts:10]: This is just a name for the HAR file.

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**